core/src/main/resources/swagger/data-repo-only.yaml

```diff
@@ -8,26 +8,26 @@ paths:
   '/api/workspaces/{workspaceNamespace}/{workspaceName}/entities':
     get:
       parameters:
-        - in: path
-          description: Workspace Namespace
-          name: workspaceNamespace
-          required: true
-          type: string
-        - in: path
-          description: Workspace Name
-          name: workspaceName
+        - $ref: '#/parameters/workspaceNamespacePathParam'
+        - $ref: '#/parameters/workspaceNamePathParam'
+        - $ref: '#/parameters/dataReferenceQueryParam'
+        - $ref: '#/parameters/billingProjectQueryParam'
```

Comment:
I moved a bunch of inlined params to a centralized `parameters:` section. There's a lot of cleanup like this that could happen in the main api-docs.yaml file; that's far out of scope of this PR.